jax/lax/lax.py

```diff
@@ -4878,6 +4862,7 @@ def _select_and_gather_add_batching_rule(
   x = batching.bdim_at_front(x, x_bdim, size)
   window_dimensions = (1,) + window_dimensions
   window_strides = (1,) + window_strides
+  padding = ((0, 0),) + padding
```

Comment:
I am wondering how did this work before. Was it working only for the VALID padtype?

Reply:
Both `SAME` and `VALID` padding are identical if the window is of size `1`.